CMakeLists.txt

```diff
@@ -131,20 +135,17 @@ set(NOVELRT_EXECUTABLE_HEADERS
 )
 
 set(NOVELRT_EXECUTABLE_SOURCES
+  ${NOVELRT_SOURCES}
```

Comment:
This shouldn't be added, the main program gets the sources from linking NovelRTLib

Reply:
I don't know how, but my local copy still shows the CMakeLists from master, so not sure what caused this. I'll make sure it's revised.